ccloud-monitoring/docs/teardown.rst

```diff
@@ -0,0 +1,102 @@
+.. _ccloud-cli-tutorial-teardown:
+
+Clean up Confluent Cloud resources
```

Comment:
We can switch to ccloud-stack destroy function/script.  Plus deleting the extra cloud key.

Reply:
working on it! getting pretty close